src/ui/Windows/PasswordPolicyDlg.cpp

```diff
@@ -903,22 +926,40 @@ void CPasswordPolicyDlg::OnGeneratePassword()
   m_ex_password.SetWindowText(m_password);
   m_ex_password.Invalidate();
 
+  m_bCopyPasswordEnabled = m_password.GetLength() > 0;
+
+  // Enable/Disable Copy to Clipboard
+  m_pCopyBtn->SetBitmap(m_bCopyPasswordEnabled ? m_CopyPswdBitmap : m_DisabledCopyPswdBitmap);
+
   UpdateData(FALSE);
 }
 
 void CPasswordPolicyDlg::OnCopyPassword()
 {
+  if (!m_bCopyPasswordEnabled)
+    return;
```

Comment:
m_bCopyPasswordEnabled is enabled if the password length > 0 i.e. one has been generated. This test prevents copying an empty string and the incorrect update of the Status bar stating a password has been copied.